deploy/olm-catalog/splunk/1.0.0/splunk.v1.0.0.clusterserviceversion.yaml

```diff
@@ -171,7 +233,7 @@ spec:
                   value: splunk-operator
                 - name: RELATED_IMAGE_SPLUNK_ENTERPRISE
                   value: docker.io/splunk/splunk:8.1.3
-                image: docker.io/splunk/splunk-operator:1.0.0
+                image: splunk/splunk-operator
```

Comment:
Doesn't it require a version number too or by default it picks up the latest version?

Reply:
See https://github.com/splunk/splunk-operator/pull/312#issuecomment-827158872